src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticAssets.targets

```diff
@@ -0,0 +1,172 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Razor.StaticAssets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+
+  <!-- Targets that support static content scenarios in ASP.NET Core.
+       The main targets are:
+       * GenerateAspNetCoreStaticAssetsManifest: Creates a manifest file to use in development with
+         the paths to all the references packages and projects content roots.
+       * ResolveAspNetCoreStaticAssetsInputs: Collects all the static assets from different sources
+         * Current project.
+         * Referenced project.
+         * Referenced packages.
+  -->
+
+  <UsingTask
+    TaskName="Microsoft.AspNetCore.Razor.Tasks.GenerateAspNetCoreStaticAssetsManifest"
+    AssemblyFile="$(RazorSdkBuildTasksAssembly)"
+    Condition="'$(RazorSdkBuildTasksAssembly)' != ''" />
+
+  <PropertyGroup>
+    <GenerateAspNetCoreStaticAssetsManifestDependsOn>
+      ResolveAspNetCoreStaticAssetsInputs;
+      _ResolveAspNetCoreStaticAssetsConfiguration;
+      _CreateAspNetCoreStaticAssetInputsCacheFile
+    </GenerateAspNetCoreStaticAssetsManifestDependsOn>
+    <ResolveAspNetCoreStaticAssetsInputsDependsOn>_ResolveAspNetCoreStaticAssetsConfiguration</ResolveAspNetCoreStaticAssetsInputsDependsOn>
+  </PropertyGroup>
+
+  <Target Name="_ResolveAspNetCoreStaticAssetsConfiguration">
+  
+    <PropertyGroup>
+      <_GeneratedAspnetCoreStaticAssetInputsCacheFile>$(IntermediateOutputPath)Microsoft.AspNetCore.StaticAssets.cache</_GeneratedAspnetCoreStaticAssetInputsCacheFile>
+      <_GeneratedAspNetCoreStaticAssetsDevelopmentManifest>$(IntermediateOutputPath)Microsoft.AspNetCore.StaticAssets.xml</_GeneratedAspNetCoreStaticAssetsDevelopmentManifest>
+    </PropertyGroup>
+
+  </Target>
+
+  <Target 
+    Name="_CreateAspNetCoreStaticAssetInputsCacheFile"
+    DependsOnTargets="ResolveAspNetCoreStaticAssetsInputs;_ResolveAspNetCoreStaticAssetsConfiguration">
+    
+    <ItemGroup>
+      <!-- 
+      This is the list of inputs that will be used for generating the manifest used during development. 
+      -->
+      <_ExternalAspNetCoreStaticAssetManifestEntry
+        Include="%(AspNetCoreStaticAsset.Identity)"
+        Condition="'%(SourceType)' != ''">
+          <BasePath>%(AspNetCoreStaticAsset.BasePath)</BasePath>
+          <ContentRoot>%(AspNetCoreStaticAsset.ContentRoot)</ContentRoot>
+      </_ExternalAspNetCoreStaticAssetManifestEntry>
+    </ItemGroup>
+
+    <Hash ItemsToHash="@(_ExternalAspNetCoreStaticAssetManifestEntry)">
```

Comment:
Does this actually hash the metadata? I think this just hashes the `%(Identity)`.

Reply:
Yeah, I think I can use a transform here to avoid the issue. Something like 
@(Items->'%(Identity)%(BasePath)%(ContentRoot)')

Reply:
The transform does the trick